crates/zensical/src/workflow.rs

```diff
@@ -31,11 +31,11 @@ use std::str::FromStr;
 use std::{fs, io};
 use zrx::id::{Id, Matcher};
 use zrx::scheduler::action::report::IntoReport;
+use zrx::stream::Stream;
```

Comment:
Please don't change the order of imports. Right now, we use the 2021 style edition, since rust-analyzer doesn't support 2024 yet.